src/tracks/timetrack/ui/TimeTrackMenuItems.cpp

```diff
@@ -28,12 +29,12 @@ void OnNewTimeTrack(const CommandContext &context)
    auto &project = context.project;
    auto &tracks = TrackList::Get( project );
 
-   if (*tracks.Any<TimeTrack>().begin()) {
-      AudacityMessageBox(
-         XO(
-"This version of Audacity only allows one time track for each project window.") );
-      return;
-   }
+//    if (*tracks.Any<TimeTrack>().begin()) {
+//       AudacityMessageBox(
+//          XO(
+// "This version of Audacity only allows one time track for each project window.") );
+//       return;
+//    }
```

Comment:
Please fully remove these as you've made a better solution.

Reply:
@LWinterberg  Done.